### DISCUSSION TITLE
Question: how can I combine with data?

### DISCUSSION THREAD

**Discussion by yealicjswo** [processing/p5.js-web-editor#3088] [General]
I wonder how to apply data(.csv, .xlsx.. etc) into P5.js..?? If it is possible with that, anyone can explain it how to apply it..??

**Comment by raclim** [comment]
Hi @yealicjswo! 

Were you thinking about something similar to [this Coding Train video, Working with Tabular Data](https://thecodingtrain.com/tracks/data-and-apis-in-javascript/data/1-client-side/2-tabular-data), or had the chance to look through the [p5.js Reference](https://p5js.org/reference/)? I think these two examples [Advanced Data: Load Saved Table](https://editor.p5js.org/p5/sketches/Advanced_Data:_Load_Saved_Table) and [Advanced Data: Load Saved JSON](https://editor.p5js.org/p5/sketches/Advanced_Data:_Load_Saved_JSON) could help illustrate that as well!
